backend/api/submissions/types.py

```diff
@@ -59,19 +71,21 @@ class Submission:
     conference: LazyType["Conference", "api.conferences.types"]
     title: str
     slug: str
-    elevator_pitch: Optional[str] = restricted_field()
-    abstract: Optional[str] = restricted_field()
-    speaker_level: Optional[str] = private_field()
-    previous_talk_video: Optional[str] = private_field()
-    notes: Optional[str] = private_field()
-    topic: Optional[LazyType["Topic", "api.conferences.types"]] = restricted_field()
-    type: Optional[SubmissionType] = restricted_field()
+    elevator_pitch: Optional[str] = restricted_field("elevator_pitch")
```

Comment:
Uhm do we maybe have access to the field name into `info`?

Reply:
but how do I get `info` in my decorator?

Reply:
you have it here: https://github.com/pythonitalia/pycon/pull/2499/files#diff-aacfacd937601019c5760248c509135b31e5b433a0aa89694434c5e8b634b8e1R18 😊

Reply:
mmm I tried with `info.field_name` but seems to be in camelCase... 🤔 
```
AttributeError: 'Submission' object has no attribute 'elevatorPitch'
```

Reply:
Ok i used `python_name` (somehow locally I can't see it as a property of Info object... )